karate-core/src/main/java/com/intuit/karate/Main.java

```diff
@@ -66,9 +67,15 @@
     @Parameters(split = "$", description = "one or more tests (features) or search-paths to run")
     List<String> paths;
 
-    @Option(names = {"-m", "--mock"}, description = "mock server file")
+    @Option(names = {"--mock"}, description = "mock server file")
     File mock;
 
+    @Option(names = {"-m", "--mocks"}, split = ",", description = "one or more mock server files")
```

Comment:
I'd like both `--mock` and `--mocks` to have the exact same behavior. can you check if this is possible with picocli